src/50_Samples_%26_Templates/Product.html

```diff
@@ -387,7 +387,9 @@ <h2 class="m1">Apple</h2>
    <!-- ## Image Gallery -->
   <!-- The `amp-carousel` component works very well for product image galleries. Learn how the `amp-carousel` component works [here](/advanced/image_galleries_with_amp-carousel/).
 
-  Notice how we are binding to the `slide` and `class` by using to the values of the variables `selectedSlide` and `selectedColor`. -->
+  Notice how we are binding to the `slide` and `class` by using to the values of the variables `selectedSlide` and `selectedColor`. 
+  It's not possible to set a default value for variables, for example we use the expression`(!selectedColor || selectedColor == 'green')`
```

Comment:
Ok, so if `selectedColor` was part of amp-state and it had `green` as value, does it mean we would not need to check `<ul class="show" [class]="(!selectedColor || selectedColor == 'green') ? 'show' : 'hide'"`